browser/brave_app_controller_mac.h

```diff
@@ -0,0 +1,18 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
```

Comment:
nit: These should ideally be bumped to 2023

Reply:
done